odoo/tools/config.py

```diff
@@ -61,61 +72,42 @@ def _deduplicate_loggers(loggers):
         for logger, level in dict(it.split(':') for it in loggers).items()
     )
 
-class configmanager(object):
-    def __init__(self, fname=None):
-        """Constructor.
-
-        :param fname: a shortcut allowing to instantiate :class:`configmanager`
-                      from Python code without resorting to environment
-                      variable
-        """
-        # Options not exposed on the command line. Command line options will be added
-        # from optparse's parser.
-        self.options = {
-            'admin_passwd': 'admin',
-            'csv_internal_sep': ',',
-            'publisher_warranty_url': 'http://services.openerp.com/publisher-warranty/',
-            'reportgz': False,
-            'root_path': None,
-            'websocket_keep_alive_timeout': 3600,
-            'websocket_rate_limit_burst': 10,
-            'websocket_rate_limit_delay': 0.2,
-        }
-
-        # Not exposed in the configuration file.
-        self.blacklist_for_save = set([
-            'publisher_warranty_url', 'load_language', 'root_path',
-            'init', 'save', 'config', 'update', 'stop_after_init', 'dev_mode', 'shell_interface',
-            'longpolling_port',
-        ])
+class configmanager:
+    def __init__(self):
+        self.options = {}
 
         # dictionary mapping option destination (keys in self.options) to MyOptions.
-        self.casts = {}
+        self.options_index = {}
+        self.casts = self.options_index  # deprecated
```

Comment:
In `[REF] core: ConfigCleaner(1) dedicated _build_cli` you say that this will be fully removed, it is not the case, we still have the two names making it quite confusing. I would fully rename it in the first commit to make it simple to understand.

Reply:
you are right, it is actually part of the next branch `master-ccleaner3-chainmap-juc`